common.sh

```diff
@@ -8,15 +8,15 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-IMAGE_AGENT=che-sidecar-workspace-data-sync
-IMAGE_STORAGE=che-workspace-data-sync-storage
+IMAGE_SIDECAR=che-sidecar-workspace-data-sync
```

Comment:
why not rename to new images...

```
che-async-storage-server
che-async-storage-sidecar
```

To align to the names in the CSV?

```
        - name: RELATED_IMAGE_async_storage_server
        - name: RELATED_IMAGE_async_storage_sidecar
```

Reply:
Ilya asked for old images to remain but then I talked to him offline and he said as long as we migrate operator (CSV) and server (che.properties) to the new image names, we can refactor those too.

Reply:
As Ilya is on PTO,  let's merge as is and have a followup issue for the refactoring here, in server, and in operator.